src/pages/profile/Profile.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+  IonButton, 
+  IonCol,   
+  IonContent, 
+  IonGrid, 
+  IonHeader, 
+  IonImg,   
+  IonInput,
+  IonItem, 
+  IonLabel,
+  IonPage,
+  IonRow, 
+  IonText, 
+  IonTitle, 
+  IonToolbar
+} from '@ionic/react';
+import './Profile.css';
+
+const Profile: React.FC = () => {
+    let email: string;    
+    let joinDate: Date;
+    let joinDateString: string;
+    let name: string;
+    let profilePic: string;
+    let totalDistance: number; // Currently in miles
+    let username: string;
+    // let badges;
+
+    function GetRecords (){
+        // TODO: Get information from database 
+        profilePic = "https://upload.wikimedia.org/wikipedia/commons/thumb/2/2b/Kristen_Stewart_in_2022.JPG/1280px-Kristen_Stewart_in_2022.JPG";
+        name = "Kristen Stewart";
+        username = "kStewart0409"
+        email = "stewart@pdx.edu";
+        joinDate = new Date("2015-03-25");
+        joinDateString = joinDate.toLocaleDateString();
+        totalDistance = 500;
+    }
+    
+    function DisplayRecords(){
+      GetRecords()
+      return (
+        <IonGrid>
+          <IonRow>
+            <IonCol size="auto">
+              <IonText>
+                <IonImg className = "profile_pic" src={profilePic} alt="Profile picture for the user signed in"></IonImg>
```

Comment:
Maybe update the alt text from the database?

Reply:
Can you give an example of what the alt text would be if we updated it from the database?

Reply:
"Alana Gilston's profile picture"
"Profile picture for Alana Gilston"
"Alana's profile picture"
"Profile picture for Alana"